lib/node_modules/@stdlib/stats/base/dists/uniform/entropy/src/main.c

```diff
@@ -0,0 +1,57 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/uniform/entropy.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/ln.h"
+#include <math.h>
+
+/**
+* Returns the entropy of a uniform distribution.
+*
+* @param n   minimum support
+* @param p   maximum support
+* @returns	 entropy
+*
+* @example
+* var v = entropy( 0.0, 1.0 );
+* // returns 0.0
+*
+* @example
+* var v = entropy( 4.0, 12.0 );
+* // returns ~2.079
+*
+* @example
+* var v = entropy( 1.0, -0.1 );
+* // returns NaN
+*
+* @example
+* var v = entropy( NaN, 2.0 );
+* // returns NaN
```

Comment:
One example is enough for the C files:

```suggestion
* @param n   minimum support
* @param p   maximum support
* @return     entropy
*
* @example
* var v = entropy( 0.0, 1.0 );
* // returns 0.0
```